src/constants/enums/bundle-status.ts

```diff
@@ -3,3 +3,6 @@ export enum BundleStatusEnum {
   APPROVAL_REQUIRED = 2,
   SUBMIT_FOR_POSTING = 3,
 }
+
+// TODO APIM-125: Postman collection has 0 in an example,
+// just wanted to double check this isn't a possible value?
```

Comment:
@avaitonis

Reply:
ACBS documentation mentions just these 3 values. In the past we used value 2, but at the moment all Bundles needs to be created with status 3. So not sure if 0 is possible.

I used this enum for initialBundleStatusCode, maybe enum name InitialBundleStatus would be better.

Reply:
Okay thanks @avaitonis, I've continued to assume 0 isn't possible for now and added `Initial` to the enum name